readme.txt

```diff
@@ -132,6 +132,7 @@ Please note that our support for the checkout block is still experimental and th
 * Fix - Fatal error on the customer payment page for subscription renewal orders with deleted products.
 * Fix - Misleading subscription order note on payment method change.
 * Fix - Incorrect error message when card ZIP validation fails.
+* Fix - Prevent Payment Request buttons from showing up in Composite Product pages. 
```

Comment:
You may move up this entry for 3.4.0. This position is for 3.3.0, which was released last week (Nov 18).